Arrangement_on_surface_2/doc/Arrangement_on_surface_2/Concepts/ArrangementDcel.h

```diff
@@ -179,7 +179,7 @@ returns a past-the-end iterator of the vertices in `dcel`.
 Vertex_const_iterator vertices_end() const; 
 
 /*!
-returns a const range over handles of the vertices in `dcel`.
+returns a const range (model of ConstRange) over handles of the vertices in `dcel`.
```

Comment:
I think that `ConstRange` should be quoted with backquotes, so that Doxygen interprets it as a type:
```
returns a const range (model of `ConstRange`) over handles of the vertices in `dcel`.
```